README.md

```diff
@@ -41,11 +41,12 @@ Even bazel is not required!
 
 ## Configuration
 
-You can configure dazel in two ways (or combine):
-* A .dazelrc file in the current directory.
+You can configure dazel in three ways (listed in order of lowest precedence to highest):
+* A `.dazelrc` file in the current directory.
+* A `.dazelrc.local` file in the current directory (this is for local user overrides - _should be .gitignored if used_).
```

Comment:
I think it would make more sense to have the "local" file in the homedir (so that it'll be loaded from ~/.dazelrc.
It's the more standard way of doing that.

Reply:
I had considered that but at least for our use case it seems that these local overrides may/will change between projects which use dazel.  I can change this if you feel that the local file in the `$HOME` dir makes the most sense, but perhaps it makes sense to have a few different levels of options.
Thoughts?

Reply:
I'm working on implementing this when I have a few minutes.  Standby for updates